vector/v.in.ogr/main.c

```diff
@@ -1053,13 +1053,20 @@ int main(int argc, char *argv[])
                 if (key_idx[layer] > -1 && key_idx[layer] == i)
                     continue; /* skip defined key (FID column) */
 
-		i_out++;
-                
 		Ogr_field = OGR_FD_GetFieldDefn(Ogr_featuredefn, i);
 		Ogr_ftype = OGR_Fld_GetType(Ogr_field);
 
 		G_debug(3, "Ogr_ftype: %i", Ogr_ftype);	/* look up below */
 
+            /* skip columns with unsupported data type */
+            if (Ogr_ftype == OFTBinary) {
+                G_warning(_("Column <%s> is omitted, binary data type is not supported."),
```

Comment:
Not supported data types are handled below at L1184

Reply:
Good point! And there all columns with unsupported data types may be omitted. However, that would be a break with present behaviour (keep column, but omit rows). Perhaps a new flag for “omit columns with unsupported data types”?

Reply:
OK, but isn't the binary column then included in the output table creation struct `grass_col_info` already in L1091: 
https://github.com/OSGeo/grass/blob/5d64de22416511b4a32e9b1546b5ae869cd0d48e/vector/v.in.ogr/main.c#L1091

So, I guess if we fix it in L1184, it would have to be removed from `col_info`?
How would I do that? A web-search unfortunately did not give me a straight forward solution...